### PR TITLE
Weld 2.4.5.Final, plus link-fix for 3.0.1 release.

### PR DESCRIPTION
**Pull request by manovotn**

### CODE REVIEW

**Comment by manovotn** [comment]
Hmm, if I include the link-fix for 3.0.1, ruby is showing the 3.x news post first...not sure this is intended.

**Comment by mkouba** [comment]
2017-08-11-weld-245Final.asciidoc should be 2017-**09**-11-weld-245Final.asciidoc

File: news/2017-08-11-weld-245Final.asciidoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+It's about time we delivered another regular bugfix release for Weld 2.
+And while the list of fixes isn't exhausting, some of them are quite crucial.
+
+Notable fixes and improvements:
+
+* Weld Core:
+** Revise Java 8 default methods plus proxies/sublasing behaviour (link:https://issues.jboss.org/browse/WELD-2407[WELD-2407, window="_blank"] and link:https://issues.jboss.org/browse/WELD-2405[WELD-2405, window="_blank"])
+** All POMs in Weld core project should now have correct MIME types; this corrects issues with Nexus 3 (link:https://issues.jboss.org/browse/WELD-2417[WELD-2417, window="_blank"])
+** Proxies for signed packages should now be generated in a custom packages (link:https://issues.jboss.org/browse/WELD-2402[WELD-2402, window="_blank"])
```
Comment (mkouba) [inline news/2017-08-11-weld-245Final.asciidoc]: _in a custom package_ or  _in custom packages_.

File: news/2017-08-11-weld-245Final.asciidoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+*** For Weld 3, we already have a better solution in link:https://issues.jboss.org/browse/WELD-2398[place, window="_blank"]
+* Weld SE and Servlet
+** Both, Weld SE and Weld Servlet, will now explicitly bring in dependency on jboss-classfilewriter 1.2+ (link:https://issues.jboss.org/browse/WELD-2406[WELD-2406, window="_blank"])
+* Other
+** Under the `jboss-as` directory of Weld source code, you can now make use of new profiles to create WildFly patches yourself link:https://issues.jboss.org/browse/WELD-2397[WELD-2397, window="_blank"]
+
+== WildFly Patch
+
+As usual, a link:http://download.jboss.org/weld/2.4.5.Final/wildfly-10.1.0.Final-weld-2.4.5.Final-patch.zip[patch for WildFly 10.1.0.Final, window="_blank"] is available.
+But since WildFly 11 has already reached CR stage, we have decided to also provide you with link:http://download.jboss.org/weld/2.4.5.Final/wildfly-11.0.0.CR1-weld-2.4.5.Final-patch.zip[ patch for WildFly 11.0.0.CR1, window="_blank"]
```
Comment (mkouba) [inline news/2017-08-11-weld-245Final.asciidoc]: Missing full stop.

**Pull request merged**